panda/src/pandabase/pandasymbols.h

```diff
@@ -603,7 +603,7 @@
   #define EXPTP_VRPN IMPORT_TEMPL
 #endif
 
-#if (defined(WIN32_VC) || defined(WIN64_VC)) && !defined(CPPPARSER)
+#if defined(_WIN32) && !defined(CPPPARSER)
```

Comment:
@rdb I need your help with this one; not sure what problem it's trying to solve.

Reply:
I also don't know why we forcibly inline math stuff on MSVC—I assume this is the result of some VR Studio profiling that happened a long time ago—but since `__forceinline` is an MSVC keyword, we should probably use `_MSC_VER` here, or not?